packages/babel-preset-default/package.json

```diff
@@ -27,13 +27,13 @@
 	],
 	"main": "index.js",
 	"dependencies": {
-		"@babel/core": "^7.4.5",
+		"@babel/core": "^7.8.3",
 		"@babel/plugin-proposal-async-generator-functions": "^7.2.0",
```

Comment:
It would be great to update other Babel plugins to their latest version for consistency and the best experience.

Reply:
There were `7.4.4` and `7.other`. So, I thought changing them would be sufficient. I'll fix them soon.

Reply:
if you encounter any issues we can split this into 2 PRs.